src/components/dashboard/news-item.vue

```diff
@@ -7,15 +7,24 @@ export default {
     Chips: () => import('~/components/common/chips.vue')
   },
   props: {
+    title: {
+      type: String
+    },
```

Comment:
When the prop object only has the type, you can just do the type as the value: `title: String`